packages/paste-website/src/components/table/index.tsx

```diff
@@ -44,7 +44,10 @@ export const Td = styled.td(props => ({
   },
 }));
 
-export const Th = styled(Td)(props => ({
+interface ThProps {
```

Comment:
Typescript was complaining that width wasn't a valid prop. I can _kinda_ see where it's going wrong in that the styled component is typed based on the element you give it. Seeing as this is a styled `Td` it may not be able to understand that it should extend the props from `td` element

Reply:
I think you're right.  I was able to get around it by doing
```js
const getTdStyles = (props: {theme: {}}): {} => ({
  padding: `${themeGet('space.space40')(props)} ${themeGet('space.space30')(props)}`,
  verticalAlign: 'top',
  wordWrap: 'break-word',

  '&:nth-of-type(1)': {
    paddingLeft: themeGet('space.space50')(props),
  },

  '&:last-child': {
    paddingRight: themeGet('space.space50')(props),
  },
});

export const Td = styled.td(getTdStyles);

export const Th = styled.th(getTdStyles, props => ({
  textAlign: 'left',
  fontWeight: themeGet('fontWeights.fontWeightSemibold')(props),
}));
```

Reply:
Is that in one of your PRs? But yeah that's the solution right there

Reply:
Nope just tested it locally while reviewing this PR